migrations/201803290755_add_initial_organizations.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const uuid = require('uuid/v4');
+
+module.exports = {
+  up: function(action) {
+    const organizations = [{
```

Comment:
There's no need to create seed organizations. Delete this file please

Reply:
why? I will get organizations, i need initial organizations in my db for test (?

Reply:
This migrations are not only for testing, but for production. When this gets released, it means a bunch of dummy organizations will be created in the database that will later on need manual removal.

It is better to create the seeds directly from the test. You can either do it by calling `Organization.createOrganization(params)` in the beforeAll for your organization's tests (and remember to delete the organization in the afterAll), or you can simpy use the `createTestSetup` that creates some basic sample organizations, users and role, to facilitate testing among all files.